build/utils.ts

```diff
@@ -294,16 +320,17 @@ export function postProcessSmallerHeadingIDs($) {
  *
  * @param {Document} doc
  */
-export function makeTOC(doc) {
+export function makeTOC(doc, withH3 = false) {
   return doc.body
     .map((section) => {
       if (
-        (section.type === "prose" ||
+        ((section.type === "prose" ||
           section.type === "browser_compatibility" ||
           section.type === "specifications") &&
-        section.value.id &&
-        section.value.title &&
-        !section.value.isH3
+          section.value.id &&
+          section.value.title &&
+          !section.value.isH3) ||
+        withH3
```

Comment:
@fiji-flo This is causing an empty entry at the beginning of the TOC, but you're probably aware?!

Reply:
Looks like it's fixed.